tests/pretty_format_kotlin_test.py

```diff
@@ -22,6 +24,15 @@ def undecorate_method():
         yield undecorated
 
 
+@pytest.mark.parametrize(
+    "version",
+    (_get_default_version("ktlint"),),
+)
+@pytest.mark.integration
+def test__download_kotlin_formatter_jar(ensure_download_possible, version):  # noqa: F811
```

Comment:
```suggestion
def test_download_kotlin_formatter_jar(ensure_download_possible, version):  # noqa: F811
```

Reply:
the double `_` is present because the usual unittest name template is `test_{function}{eventual_other_conditions}`
and function is `_download_kotlin_formatter_jar` :)

Reply:
Legit